jcl/src/java.base/share/classes/java/lang/J9VMInternals.java

```diff
@@ -212,10 +219,14 @@ private static void recordInitializationFailure(Class clazz, Throwable err) {
 	 * Otherwise, wrap it in a new ExceptionInInitializerError and throw that.
 	 */
 	private static void ensureError(Throwable err) {
+/*[IF JAVA_SPEC_VERSION >= 18]*/
+		throw (Error)err;
+/*[ELSE] JAVA_SPEC_VERSION >= 18*/
 		if (err instanceof Error) {
 			throw (Error)err;
 		}
 		throw new ExceptionInInitializerError(err);
```

Comment:
@babsingh Perhaps this is still necessary if `recordInitializationFailure` skips the `if (initialized)` block

Reply:
yes, it's possible. let's keep it.